doc/classes/MultiMesh.xml

```diff
@@ -68,6 +68,15 @@
 				[Transform] is stored as 12 floats, [Transform2D] is stored as 8 floats, [code]COLOR_8BIT[/code] / [code]CUSTOM_DATA_8BIT[/code] is stored as 1 float (4 bytes as is) and [code]COLOR_FLOAT[/code] / [code]CUSTOM_DATA_FLOAT[/code] is stored as 4 floats.
 			</description>
 		</method>
+		<method name="get_as_bulk_array">
+			<return type="PoolRealArray">
+			</return>
+			<description>
+				Gets all data related to the instances in one go. This is especially useful when loading the data to disk or processing the data in GDNative.
```

Comment:
The description needs to be updated to be very clear that this will cause stalls in multithreaded environments. Also, the first line is not really applicable anymore. The use case for set_as_bulk_array is very different

Reply:
I removed the second sentence and replaced it with "Note that this will cause stalls in multithreaded environments". So the whole first line is `Gets all data related to the instances in one go. Note that this will cause stalls in multithreaded environments.` and the rest is the same. Does that work?

Both VisualServer.xml and MultiMesh.xml have that same text.